docs/rst/types/data_kind.rst

```diff
@@ -5,8 +5,93 @@
 DataKind
 ========
 
-Work in progress
+The *eProsima Fast DDS Statistics Backend* records statistics data of different
+nature, as provided by *eProsima Fast DDS*, e.g., latency or message count.
+We refer to the nature of these data values as their |DataKind-api|.
 
-.. todo::
+- |FASTDDS_LATENCY-api|: The latency between a write operation
+  in the data writer and the moment the data is available in the data reader
+
+- |NETWORK_LATENCY-api|: The Latency in the communication between two locators
+
+- |PUBLICATION_THROUGHPUT-api|: Amount of data (in Mb/s) sent by a data writer
+
+- |SUBSCRIPTION_THROUGHPUT-api|: Amount of data (in Mb/s) received by a data reader
+
+- |RTPS_PACKETS_SENT-api|: Amount of packets sent from a participant to a Locator
```

Comment:
```suggestion
- |RTPS_PACKETS_SENT-api|: Amount of packets sent from a participant to a locator
```

This applies to other `Locator`. The entities are not with capital letter so I do not think locator should either.